CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## Unreleased
+- Kotlin 1.6.10
+- Gradle 7.3.2
+- Coroutines 1.5.2
+- Added refreshVersions plugin
+- New redux-kotlin-compose module
```

Comment:
lots of changes for 1 commit.  
Ideally these are broken into their own commits.